docs/zkapps/lightnet.mdx

```diff
@@ -0,0 +1,214 @@
+- Launches the lightweight Mina Explorer in a web browser so you can monitor transactions.
+
+:::info
+
+Use of Lightnet blockchain is appropriate for local development and testing only. It is not intended to replicate all aspects of the full network.
+
+:::
+
+## Prerequisites
+
+Docker Engine is required for your environment. 
+
+1. [Install Docker Engine](
+https://docs.docker.com/engine/install/).
+1. Start Docker Engine. 
+
+Docker Engine must be running before you can start the Docker container.
+
+## Start a single node network
+
+To start a single node network with default behavior:
+
+```sh
+zk lightnet start
+```
+
+This command uses default settings that successfully serve the majority of testing requirements:
+
+- Pulls the latest Docker image for your environment from the [Docker Hub](https://hub.docker.com/r/o1labs/mina-local-network) repository
+- Prepares the file system
+- Downloads and updates the lightweight Mina Explorer
+- Forms the network with a single node
+- Configures the network to be fast
+- Disables the blockchain SNARK proofs
+- Uses the `o1js-main` branch of the Mina repository
+- Starts the Mina archive process and the archive node API
+- Waits for the network to reach a synchronized state
+- Sets the Mina process logging level to trace
+
+## Start a local network with other settings
+
+To see the options to start a local blockchain network with non-default settings:
+
+```sh
+zk lightnet start --help
+```
+
+You can configure different network properties. For example, to start a multi-node local blockchain network with slower, close-to-real-world properties:
+
+```sh
+zk lightnet start --mode multi-node --type real
+```
+
+## Keep current versions
+
+New Docker images are built and published to the Docker Hub every night. You might not always want to get the latest product versions. For example, when your zkApp relies on well-defined APIs and you want to continue developing in your current environment. 
+
+To keep your current working versions of o1js and Mina:
+
+```sh
+zk lightnet start --no-pull
+```
+
+## Stop the local network
+
+To stop the local network, remove the Docker container, and clean up the environment:
+
+```sh
+zk lightnet stop
+```
+
+When Lightnet is stopped, the log files for Docker container services are saved to the host file system at `/Users/<username>/.cache/zkapp-cli/lightnet/logs/`.
+
+To disable saving of log files:
+
+```sh
+zk lightnet stop --no-save-logs
+```
+
+## Troubleshooting transactions
+
+Tools are provided for troubleshooting.
+
+### Lightweight Mina Explorer
+
+To monitor transactions, launch the lightweight Mina Explorer:
+
+```sh
+zk lightnet explorer
+```
+
+By default, the latest version is launched. 
+
+To list versions, their published dates, and show the version in use:
+
+```sh
+zk lightnet explorer list true
+```
+
+To use a specific version of the lightweight Mina Explorer:
+
+```sh
+zk lightnet explorer use <version>
+```
+
+### Reinstantiate lightnet
```

Comment:
Section missing?

Reply:
there is no instantiate command, so I have removed. Love this eagle eye feedback, thank you